### PR TITLE
ci: Add integrtion tests for azblob

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

### CODE REVIEW

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -145,3 +145,66 @@ jobs:
...
+          /home/runner/.cargo/bin/sccache --show-stats | grep azblob
+
+      - name: Test Twice for Cache Read
+        run: cargo clean && cargo build
+        env:
+          RUST_BACKTRACE: full
+          RUST_LOG: debug
+          RUSTC_WRAPPER: /home/runner/.cargo/bin/sccache
+          SCCACHE_AZURE_BLOB_CONTAINER: "test"
+          SCCACHE_AZURE_CONNECTION_STRING: "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;"
```
Comment (sylvestre) [inline .github/workflows/integration-tests.yml:204]: Can the login and password be moved into a single place? Thanks

Comment (Xuanwo) [reply .github/workflows/integration-tests.yml:204]: What's the meaning of `moving into a single place`?

Comment (sylvestre) [reply .github/workflows/integration-tests.yml:204]: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1; 

is duplicated 3 times in this file ;)

Comment (Xuanwo) [reply .github/workflows/integration-tests.yml:204]: I got it~

**Comment by Xuanwo** [comment]
@sylvestre you must be happy with the current implementation! :laughing:

**Comment by sylvestre** [comment]
indeed, many thanks

**Pull request merged**